src/acl/Acl.h

```diff
@@ -23,6 +23,8 @@ class ConfigParser;
 
 namespace Acl {
 
+class Answer;
```

Comment:
This predefine should be (and is already) done by acl/forward.h instead. The other code in here should be too, but out of scope for this PR.

So for here, just remove the 2 new lines.